README.md

```diff
@@ -79,10 +79,24 @@ Docker or on top of [Kubernetes].
 
 ## Getting Started
 
-You can start off with
+You can start utilising Authelia with the provided `docker-compose` bundles:
 
-    git clone https://github.com/authelia/authelia.git && cd authelia
-    source bootstrap.sh
+##### Local
```

Comment:
In order to avoid maintaining twice the same content, I'd rather redirect the user to the Getting Started documentation directly. If we really want to keep some explanations here, which would make sense, that would be something like:

```
- `git clone https://github.com/authelia/authelia.git`
- `cd authelia/compose/local`
- `docker-compose up -d`

Use user `john` with password `password` to visit `https://traefik.example.com` protected with two factors or `https://singlefactor.example.com` protected with only one factor.
```